### ISSUE TITLE
Ampersand, html encoding

### ISSUE THREAD

**Issue by lastmjs** [kcmr/code-sample#20]
I'm running into a little issue here:

This:

```html
<code-sample>
  <template>
    function both(x, y) {
      if (x && y) {
        return 'Both';
      }
    
      if (x && !y) {
        return 'First';
      }
    
      if (!x && y) {
        return 'Second';
      }
    }
  </template>
</code-sample>
```

renders like this:

```javascript
function both(x, y) {
  if (x &amp;&amp; y) {
    return 'Both';
  }
    
  if (x &amp;&amp; !y) {
    return 'First';
  }
    
  if (!x &amp;&amp; y) {
    return 'Second';
  }
}
```

Do you know how I can easily get ampersands to display properly?

**Comment by lastmjs** [comment]
Also, using `&amp;` instead of `&` does not fix the issue.

**Comment by kcmr** [comment]
Hi @lastmjs 
I've just published 3.0.2 which solves the issue. 
Thank you for reporting it!
